examples/tv-casting-app/android/App/app/src/main/java/com/matter/casting/ConnectionExampleFragment.java

```diff
@@ -152,9 +228,109 @@ public void handle(MatterError err) {
             });
   }
 
+  private void displayPasscodeInputDialog(Context context) {
+    AlertDialog.Builder builder = new AlertDialog.Builder(context);
+
+    String title = "Enter the Commissioner-Generated Passcode";
+    String instructions =
+        "Input the Commissioner-Generated passcode displayed on the CastingPlayer UX, or use the default provided (12345678).";
+
+    LayoutInflater inflater = LayoutInflater.from(context);
+    View dialogView = inflater.inflate(R.layout.custom_passcode_dialog, null);
+
+    TextView titleTextView = dialogView.findViewById(R.id.dialog_title);
+    TextView instructionsTextView = dialogView.findViewById(R.id.dialog_instructions);
+    titleTextView.setText(title);
+    instructionsTextView.setText(instructions);
```

Comment:
These should be already added to the xml layout in custom\_passcode\_dialog.xml, without needing to add it programatically here.

Reply:
right, removing.